modules/migration-debugging-velero-resources.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/troubleshooting-3-4.adoc
+// * migration/migrating_4_1_4/troubleshooting-4-1-4.adoc
+// * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
+
+[id="migration-debugging-velero-resources_{context}"]
+= Using the Velero CLI to debug Backup and Restore CRs
+
+You can debug the `Backup` and `Restore` custom resources (CRs) and partial migration failures with the Velero command line interface (CLI). The Velero CLI runs in the `velero` pod.
+
+== Velero command syntax
```

Comment:
Each of the level 2 headings must have a unique section ID associated with it (Velero command syntax, Help command, etc.)

Reply:
Oops -- yes, I forgot to do that. Will fix.